app/build.gradle

```diff
@@ -12,7 +12,7 @@ def getVersionName = { ->
 
 android {
     compileSdkVersion 26
```

Comment:
Why not set this to 28 as well?

Reply:
What implications did you see? I didn't look very closely, but didn't noticed anything that might impact us...

Reply:
Sorry, I usually update the `compileSdkVersion` together with the `targetSdkVersion`, so was actually referring to updating both. It compiles fine with 28 so updating the `compileSdkVersion` should be safe.

We might also want to set the `targetSdkVersion` to 28 as we just made a new release, there should be enough time to test if everything still works as intended.